web-client/src/components/Buttons/index.tsx

```diff
@@ -70,3 +70,35 @@ export const StepForwardButton = styled(Button)`
     color: white;
   }
 `;
+
+export const SettingsButton = styled(Button)`
+  display: flex;
+  justify-content: start;
+  font-size: 20px
+  font-weight: normal;
+  padding: 10px;
+  width: 100%;
+  > span {
+      display: flex;
+      align-items: center;
+      justify-content: space-around;
+  }
+`;
+
+export const ModalCancelButton = styled(Button)`
```

Comment:
Change this name or figure out somewhere else to put this.  There are too many different modals.   This is the CancelButton for which of the three modals?